src/PKSim.Presentation/Services/ProjectTask.cs

```diff
@@ -213,6 +217,35 @@ public void Run(StartOptions startOptions)
          }
       }
 
+      public void LoadProjectFromSnapshot()
+      {
+         if (!shouldCloseProject()) return;
+
+         var message = PKSimConstants.UI.LoadFromSnapshotFile(PKSimConstants.ObjectTypes.Project);
+         var fileName = _dialogCreator.AskForFileToOpen(message, Constants.Filter.JSON_FILE_FILTER, Constants.DirectoryKey.REPORT);
+         if (string.IsNullOrEmpty(fileName))
+            return;
+
+         _heavyWorkManager.Start(() => _workspace.LoadProject(ProjectFromSnapshot), PKSimConstants.UI.LoadingProject);
+
+         void ProjectFromSnapshot()
+         {
+            var project = _snapshotTask.LoadFromSnapshot<PKSimProject>(fileName).FirstOrDefault();
+            if (project == null)
+               return;
+
+            closeProject();
+            _workspace.Project = project;
+            project.HasChanged = true;
```

Comment:
Setting to true explicitly here since we do not load from a project file. Question should be asked to the user